src/policy/packages.h

```diff
@@ -15,18 +15,25 @@
 
 /** Default maximum number of transactions in a package. */
 static constexpr uint32_t MAX_PACKAGE_COUNT{25};
-/** Default maximum total virtual size of transactions in a package in KvB. */
-static constexpr uint32_t MAX_PACKAGE_SIZE{101};
-static_assert(MAX_PACKAGE_SIZE * WITNESS_SCALE_FACTOR * 1000 >= MAX_STANDARD_TX_WEIGHT);
+/** Default maximum total weight of transactions in a package in weight
+    to allow for context-less checks. This includes room for weight to vsize rounding error
+    for all included package transactions to not block transactions that would have all
+    been accepted individually. */
+static constexpr uint32_t MAX_PACKAGE_WEIGHT = 404'000 +  MAX_PACKAGE_COUNT * (WITNESS_SCALE_FACTOR - 1);
```

Comment:
In 1f25d0e629820a885a9425d8e0115087f8183cb3 "Replace MAX_PACKAGE_SIZE with MAX_PACKAGE_WEIGHT to avoid vbyte confusion"

Since weight to vsize is rounded up, this limit does not need to move up. The total weight will never be more than `vsize * WITNESS_SCALE_FACTOR`. So 404,000 weight units is the correct conversion here.

Reply:
done